README.md

```diff
@@ -8,7 +8,10 @@ We conduct ongoing audits that review all changes to the code to ensure a high d
 
 |  Date   | Audit                                                                                 |                                                       Mitigations                                                       | Scope                                                                 |
 | :-----: | :------------------------------------------------------------------------------------ | :---------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------- |
+| 2024-11 | [NCC](audits/2024-11%20NCC.pdf)                           |  | `interchain-token-service` `axelar-amplifier`
+| 2024-11 | [Ackee](audits/2024-11%20Ackee%20Blockchain.pdf)                           |  | `interchain-token-service`
 | 2024-11 | [Ottersec](audits/2024-11%20Ottersec%20-%20Sui.pdf)                           | [Here](https://bright-ambert-2bd.notion.site/15bc53fccb7780d48608d3657b6662ec?v=a804f3291a1a4cd58a7a6ab633ca85b0&pvs=4) | `axelar-cgp-sui`
+| 2024-08 | [Code4rena](https://github.com/code-423n4/2024-08-axelar-network)                           |  | `interchain-token-service`
```

Comment:
I think we should add `axelar-amplifier` as well, https://github.com/code-423n4/2024-08-axelar-network?tab=readme-ov-file#files-in-scope